src/mozilla_sec_eia/ex_21/rename_labeled_filings.py

```diff
@@ -20,15 +22,60 @@ def rename_filings():
     filename.
     """
     archive = GCSArchive()
-    bucket = archive._filings_bucket
+    bucket = archive._labels_bucket
 
-    labeled_bucket_name = "labeled"
+    labeled_bucket_name = "labeled/"
 
     for blob in bucket.list_blobs(prefix=labeled_bucket_name):
-        if blob.name != labeled_bucket_name + "/":
+        if blob.name != labeled_bucket_name:
             logger.info(blob.name)
             file_dict = json.loads(blob.download_as_text())
             archive_name = file_dict["task"]["data"]["ocr"].split("/")[-1].split(".")[0]
             archive_filepath = f"{labeled_bucket_name}/{archive_name}"
             logger.info(archive_filepath)
             bucket.rename_blob(blob, archive_filepath)
+
+
+def copy_labeled_jsons_to_new_version_folder(
```

Comment:
This is a kind of messy helper function to move training data from one version's GCS folder to another. For example, I made version 0.1 which combined multi page documents into one image. However, single page documents were unchanged and I didn't want to relabel them, so I moved them from the v0.0 folder into the v0.1 folder if they didn't already exist in the destination folder.